offlinedocs/pages/_app.tsx

```diff
@@ -0,0 +1,29 @@
+import { ChakraProvider, extendTheme } from "@chakra-ui/react"
```

Comment:
We usually use 2 space indentation for ts and tsx files, this appears to be using tabs.

The other files in this PR seem to also be using tabs

Reply:
Good catch, I will fix that.